.github/workflows/docker-commit.yaml

```
@@ -38,6 +38,8 @@ jobs:
       - uses: azure/container-scan@v0
         with:
           image-name: ${{ env.REGISTRY }}/${{ env.GITHUB_REPOSITORY_OWNER_PART_SLUG }}/${{ env.GITHUB_REPOSITORY_NAME_PART_SLUG }}:${{ env.GITHUB_REF_SLUG }}-${{ github.sha }}
+        env:
+          DOCKLE_HOST: "unix:///var/run/docker.sock"
```

Dockel? 

```suggestion
          DOCKER_HOST: "unix:///var/run/docker.sock"
```

Also is this not the default value already?